examples/workspace_access.tf

```diff
@@ -0,0 +1,15 @@
+data "prefect_workspace_role" "developer" {
+	name = "Developer"
+}
+data "prefect_workspace" "prd" {
+	id = "<workspace uuid>"
+}
+resource "prefect_service_account" "bot" {
+	name = "a-cool-bot"
+}
+resource "prefect_workspace_access" "bot_access" {
+	accessor_type = "SERVICE ACCOUNT"
```

Comment:
should this be `SERVICE_ACCOUNT` as seen below in 
```
const ServiceAccount string = "SERVICE_ACCOUNT"
```

Reply:
indeed it should, good catch